09_json/n0npax/test-data/data.json

```diff
@@ -0,0 +1,6 @@
+[
+{"id":0,"value":4,"breed":"Type: D","colour":"Red"},
+{"id":0,"value":3,"breed":"Type: U","colour":"White"},
+{"id":0,"value":2,"breed":"Type: P","colour":"Green"},
+{"id":0,"value":1,"breed":"Type: A","colour":"Blue"}
+]
```

Comment:
The specs for the labs says `Test marshalling and unmarshalling using require.JSONEq`
I only see you tested marshal, and not `unmarshal`. Also you didn't use `require.JSONEq`

Reply:
good point. Will be fixed

Reply:
FIXED. thanks